src/markdown-script.ts

```diff
@@ -1,14 +1,46 @@
-/**
- * This script runs in VSCode markdown builtin preview webview
- */
+
 import { pintoraStandalone } from '@pintora/standalone'
 
 let hasThemeBeSet = false
+const openedErrors = {}
+
+const addCSS = css => (document.head.appendChild(document.createElement('style')).innerHTML = css)
+addCSS(`
+  .pintora-error pre {
+   color: var(--vscode-errorForeground);
+  }
+  .pintora-error pre {
+    display: none;
+  }
+  .pintora-error.open pre {
+   display: block;
+   color: #ff4c4c;
+  }
+  .toggle {
+   margin-top: 0.2em;
+   display: block;
+   font-weight: bold;
+   font-family: monospace;
+   font-size: 1.2rem;
+   padding: 1rem;
+   color: #ff4c4c;
+   cursor: pointer;
+   border-radius: 7px;
+   transition: all 0.25s ease-out;
+   background: #f553;
```

Comment:
Does VSCode has variable like errorBackground like the `--vscode-errorForeground` above, so this css can adapt well to all the themes ?

Just a tiny thought. I can try digging it later after this PR is merged.

Reply:
Oohh, I will research it.

Reply:
Do you think its this one? ```--vscode-inputValidation-errorBackground ```

![image](https://user-images.githubusercontent.com/106539458/187057207-02e183ad-fe3f-469c-8d55-7235c0210c58.png)
![image](https://user-images.githubusercontent.com/106539458/187057228-08df3418-c5b6-40d2-b734-d5186678f599.png)

Reply:
https://code.visualstudio.com/api/references/theme-color
I find no one better than this, so let's keep this.